otel/conf.go

```diff
@@ -33,7 +33,14 @@ var confTemplate = template.Must(template.New("conf").Parse(
   {{template "iis" .}}
 {{- end}}
 processors:
-  {{template "defaultprocessor" .}}
+  {{template "systemprocessor" .}}
+{{- if .IIS}}
+  {{template "iisprocessor" .}}
+{{- end}}
+{{- if .MSSQL}}
+  {{template "mssqlprocessor" .}}
+{{- end}}
+  {{template "agentprocessor" .}}
```

Comment:
It would make sense to move this line, so that it immediately follows `{{template "systemprocessor" .}}`. 

I opted not to make this change yet, so as to minimize changes to the golden configs.